api/tests/c/indigo-test-issue-349.cpp

```diff
@@ -0,0 +1,107 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "indigo.h"
+
+const char* CID58374005TestData = R"(CID 58374005 edited
```

Comment:
in future, if you use cpp tests the molfiles worth to add into separate text files. There are warning  messages regarding this lines in gcc compiler

Reply:
right now please remove the tests (and move everything in python)

Reply:
Done